kratos/processes/find_nodal_h_process.h

```diff
@@ -46,6 +46,16 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
+/**
+ * @brief This struct is used in order to identify when using the hitorical and non historical variables
+ */
+struct FindNodalH
+{
+    // Defining clearer options
+    constexpr static bool SaveAsHistoricalVariable = true;
+    constexpr static bool SaveAsNonHistoricalVariable = false;
+};
+
```

Comment:
I think that @pooyan-dadvand suggested the direct use of 2 constexpr, not inside a struct.
Do you agree?

Reply:
I tried, and does not work

Reply:
ok then.

Reply:
Exactly, I suggest two constexpr in the same class. Why doesn't work?

Reply:
Solved in #3331